sdk-ingestion/src/main/java/care/data4life/sdk/Data4LifeClient.java

```diff
@@ -128,7 +131,18 @@ public static Data4LifeClient init(byte[] accessToken,
         NetworkConnectivityService networkConnectivityService = () -> true;
 
         // Create ApiService that uses a static token
-        ApiService apiService = new ApiService(authorizationService, environment, clientId, DUMMY_CLIENT_SECRET, platform, networkConnectivityService, CLIENT_NAME, accessToken, DEBUG);
+        ApiService apiService = new ApiService(
+                authorizationService,
+                environment,
+                clientId,
+                DUMMY_CLIENT_SECRET,
+                platform,
+                networkConnectivityService,
+                NetworkingContract.Clients.INGESTION,
+                VERSION,
```

Comment:
The version is now static, but it is controlled by a Git tag, Idealy it is injected into the codebase.

Reply:
You are totally right - My question is though, how we get it similar done like in the Android client/make the BuildConfig available also for JVM?

Reply:
As discussed in other place - I will provide a follow up to make this dynamic.